opentelemetry-api/src/opentelemetry/util/_time.py

```diff
@@ -0,0 +1,39 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from logging import getLogger
+from sys import version_info
+
+if version_info.minor < 7:
```

Comment:
IMO a safer way to do this is:

```
try:
    from time import time_ns
except ImportError:
   def _time_ns....
```

This avoids anything funky with backported functions and the like.

Reply:
Sorry, how can this avoid anything funky with backported functions? I prefer the `if` approach because it is not-exceptional for our users to use 3.6 or 3.5 so it should not be treated as an exception.

Reply:
```
Sorry, how can this avoid anything funky with backported functions?
```
In the past, people have [backported python features back to](https://pypi.org/project/functools32/) [older versions of python](https://pypi.org/project/typing/). It's common for them to use the same module name, or modify an existing module to include the functionality they backported.

There's no guarantee that the version of python doesn't have some version of the function backported. Hence I was saying that it's more common to just check to see if the import fails, and if it doesn't, then import the function.

Reply:
Ok, but we are not adding anything to any existing module now, I mean, we no longer try to patch `time`. `_time_ns` is always an attribute of `opentelemetry.util` regardless of the Python version used, only its implementation changes.

Reply:
I'm more concerned about anyone patching the module. In the examples above, someone else could add time_ns, and if it exists, it would be nice to do so. 

But this isn't a blocker comment, practically I don't think this will make much of a difference.

Reply:
@toumorokoshi 
Is this resolved?